lib/resque.rb

```diff
@@ -271,11 +271,23 @@ def push(queue, item)
     data_store.push_to_queue(queue,encode(item))
   end
 
-  # Pops a job off a queue. Queue name should be a string.
+  # Pops a job off a set of queues. Queue names should be a string.
```

Comment:
`Queue names should be a string.` That comment seems untrue, your test case below uses symbols, so let's drop it for sanity.

Reply:
True, I have adjusted the original comment. We could change it.